### ISSUE TITLE
Can someone upload a complete zip repo ?

### ISSUE THREAD

**Issue by Michaelwhite34** [NVlabs/instant-ngp#892]
For some reason I always fail to get the full dependency with git clone  --recursive or submodule update --init --recursive. 
And I am getting error in cmake --build build --config RelWithDebInfo -j with "No such file or directory [E:\deep_learning_stuff\New_Folder\ngp\instant-ngp\build\optix_program.vcxproj]"

**Comment by jbb-kryo** [comment]
You are missing prerequisites. This is not a fault with the repo. It is an issue with your installed prerequisites.

Ensure you have installed the following prerequisistes:
- [CUDA](https://developer.nvidia.com/cuda-toolkit)
- [Python 9/10](https://www.python.org/)
- [Optix](https://developer.nvidia.com/optix) ---> _You are missing this one specifically._
- [Vulkan SDK](https://vulkan.lunarg.com/)
- [Anaconda](https://www.anaconda.com/products/distribution)

**Comment by Michaelwhite34** [comment]
> You are missing prerequisites. This is not a fault with the repo. It is an issue with your installed prerequisites.
> 
> Ensure you have installed the following prerequisistes:
> 
> * [CUDA](https://developer.nvidia.com/cuda-toolkit)
> * [Python 9/10](https://www.python.org/)
> * [Optix](https://developer.nvidia.com/optix) ---> _You are missing this one specifically._
> * [Vulkan SDK](https://vulkan.lunarg.com/)
> * [Anaconda](https://www.anaconda.com/products/distribution)

No, I solved the problem by recloning some many times to get the complete repo.
